atlas/AtlasMl/atlasml/clients/weaviate.py

```
@@ -1,39 +1,88 @@
+            {"name": "cluster_id", "data_type":  DataType.UUID, "indexFilterable": True},
+            {"name": "label_id", "data_type": DataType.TEXT, "indexFilterable": True},
+        ]
+    },
+}
+
 
 logger = logging.getLogger(__name__)
 
 
+
+class WeaviateConnectionError(Exception):
```

Ruff (0.12.2)

Redefinition of unused `WeaviateConnectionError` from line 9

(F811)